mopidy_pandora/library.py

```diff
@@ -127,11 +129,39 @@ def get_images(self, uris):
         for uri in uris:
             image_uris = set()
             try:
-                track = self.lookup_pandora_track(uri)
-                if track.is_ad is True:
-                    image_uri = track.image_url
+                image_uri = None
```

Comment:
I would suggest to split this into a separate function, as the amount of indentation goes up to 7.